cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -134,6 +134,15 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+# Wait for the default service account
+token_found=""
+while [ -z "${token_made}" ]; do
+  sleep .5
+  token_found=$(${KUBECTL} get serviceaccount default -o template -t "{{with index .secrets 0}}{{.name}}{{end}}" || echo "")
```

Comment:
I suspect it was an effort to ensure that the output was empty (since the pipeline on error distills to `$(false || echo)`), but this isn't necessary, `true` also has empty output.